### ISSUE TITLE
weight conversion tool

### ISSUE THREAD

**Issue by shubham2704** [codeezzi/utility#2]
#  Create a weight conversion tool in html,css and javascript.

- Add navbar in your tool's html file. You can copy the navbar from index.html.
- Add screenshot of your tool in, PR message and also in the folder of your tool
- Before Making the PR Pull the latest changes that have been made to the repository

**Note: Please Don't Make Any Changes In `index.html` AND `contributors.html` **
--

## I have;

- [ ] Read contibuting.md
- [ ] Followed all the guidelines
- [ ] updated readme.md

**Comment by prakhar-ai** [comment]
Can I work on this issue?

**Comment by shubham2704** [comment]
@prakhar-ai are you still working on this?

read the new guidelines:
- Add navbar in your tool's html file. You can copy the navbar from index.html.
- Add screenshot of your tool in, PR message and also in the folder of your tool
- Before Making the PR Pull the latest changes that have been made to the repository

**Note: Please Don't Make Any Changes In `index.html` AND `contributors.html` **

**Comment by prakhar-ai** [comment]
@shubham2704 Yes, I'm sorry for the delay. Will be adding the pull request in a few hours.

**Comment by shubham2704** [comment]
@prakhar-ai No problem! As our contributing guidelines have been changed so read them before and make changes accordingly before makin the pull request. ✌

**Comment by shubham2704** [comment]
@prakhar-ai Are you still working on this issue.

**Comment by ramonborges15** [comment]
@shubham2704 can you assign it to me?

**Comment by programmerraja** [comment]
@shubham2704 
Is this still open i completed this work sir
Can i pull my request

**Comment by shubham2704** [comment]
@programmerraja Yes you can crete a new pull request for this issue. ✌

**Comment by programmerraja** [comment]
@shubham2704  sir can you take look my pull request #135 
Just 3 days to go sir to complete hacktoberfest
